docs/intro.rst

```diff
@@ -207,7 +207,24 @@ Mapping
 
 
 Enum
-....
+----
+
+Enum object can contain values of different types. Enum check determines whether given value is in the Enum instance.
+
+Example::
+
+  >>> Enum(1, 2, 'error').check(2)
+  2
```

Comment:
U can see that now in documentation all objects from trafaret use with `t`prefix.

```python
t.Bytes
```

Let's follow this style.

Reply:
Ok. Should i provide the context by adding import of trafaret to the example? Or just added prefix will be ok?

Reply:
no only the prefix. U can use other documentation in this file like example.

Reply:
Ok